catroid/src/main/java/org/catrobat/catroid/content/BroadcastListener.java

```diff
@@ -25,29 +25,16 @@
 import com.badlogic.gdx.scenes.scene2d.Event;
 import com.badlogic.gdx.scenes.scene2d.EventListener;
 
-import org.catrobat.catroid.content.BroadcastEvent.BroadcastType;
-
-public class BroadcastListener implements EventListener {
+public abstract class BroadcastListener implements EventListener {
 
 	@Override
 	public boolean handle(Event event) {
-		if (event instanceof BroadcastEvent) {
-			BroadcastEvent broadcastEvent = (BroadcastEvent) event;
-			if (broadcastEvent.getType().equals(BroadcastType.broadcast)) {
-				handleBroadcastEvent(broadcastEvent, broadcastEvent.getBroadcastMessage());
-				return true;
-			}
-			if (broadcastEvent.getType().equals(BroadcastType.broadcastWait)) {
-				handleBroadcastFromWaiterEvent(broadcastEvent, broadcastEvent.getBroadcastMessage());
-				return true;
-			}
+		if (event instanceof EventWrapper) {
```

Comment:
.

Reply:
Touch events are among the events that should be ignored.